pydicom/fileset.py

```diff
@@ -1508,34 +1508,40 @@ def find_values(
 
         Returns
         -------
-        list of object
-            A list of value(s) for the element available in the instances.
+        list of object(s), or dict of lists of object(s)
+            If single element was queried: A list of value(s) for the element
+                available in the instances.
+            If list of elements was queried: A dict of element value pairs with
+                lists of value(s) for the elements available in the instances.
```

Comment:
Should be like this so the [formatting](https://2941-14006067-gh.circle-artifacts.com/0/doc/_build/html/reference/generated/pydicom.fileset.FileSet.html#pydicom.fileset.FileSet.find_values) is correct:
```
list of object(s), or dict of lists of object(s)

    * If single element was queried: A list of value(s) for the element
      available in the instances.
    * If list of elements was queried: A dict of element value pairs with
      lists of value(s) for the elements available in the instances.
```